lib/propmangle.js

```diff
@@ -81,18 +82,21 @@ function mangle_properties(ast, options) {
     }
 
     var regex = options.regex;
+    var ignore_quoted = options.ignore_quoted;
 
     var names_to_mangle = [];
     var unmangleable = [];
 
     // step 1: find candidates to mangle
     ast.walk(new TreeWalker(function(node){
         if (node instanceof AST_ObjectKeyVal) {
-            add(node.key);
+            if (!(ignore_quoted && node.quote))
+                add(node.key);
         }
         else if (node instanceof AST_ObjectProperty) {
             // setter or getter, since KeyVal is handled above
-            add(node.key.name);
+            if (!(ignore_quoted && node.key.quote))
```

Comment:
This `if` should be removed as `AST_ObjectProperty` does not have a `quote` property.